application/src/main/java/org/togetherjava/tjbot/commands/tags/TagSystem.java

```diff
@@ -57,17 +72,40 @@ static Button createDeleteButton(String componentId) {
      * @param event the event to send messages with
      * @return whether the given tag is unknown to the system
      */
-    boolean handleIsUnknownTag(String id, IReplyCallback event) {
+    boolean handleIsUnknownTag(String id, IReplyCallback event,
+            Function<String[], String> componentIdGenerator,
+            @Nullable OptionMapping userToReplyTo) {
```

Comment:
what is this? also, missing documentation. ill recheck ur code for this once u added doc.

Reply:
recheck please